src/btree/col_srch.c

```diff
@@ -230,14 +230,17 @@ __wt_col_search(
     if (page->type == WT_PAGE_COL_FIX) {
         if (recno < current->ref_recno) {
             cbt->recno = current->ref_recno;
+            cbt->slot = 0;
             cbt->compare = 1;
             return (0);
         }
         if (recno >= current->ref_recno + page->entries) {
             cbt->recno = current->ref_recno + page->entries;
+            cbt->slot = 0;
             goto past_end;
         } else {
             cbt->recno = recno;
+            cbt->slot = 0;
```

Comment:
I don't remember exactly -- I think because of the code we put in __wt_read_cell_time_window that checks the slot for UINT32_MAX I wanted to make sure it's set to UINT32_MAX if the cursor isn't positioned, and then it fails if you don't set it back to 0. Or something like that. Anyway, I think it's good for the slot to be set or unset in accordance with the other page types, even though it can only be set to 0.

Reply:
If/when you look at `cbt.slot` more generally, it might be best to clear it in `__cursor_pos_clear()`.

Reply:
Noted...